pkg/types/ovirt/platform.go

```diff
@@ -23,9 +23,6 @@ type Platform struct {
 	// APIVIP is an IP which will be served by bootstrap and then pivoted masters, using keepalived
 	APIVIP string `json:"api_vip"`
 
-	// DNSVIP is the IP of the internal DNS which will be operated by the cluster
```

Comment:
can you run https://github.com/openshift/installer/blob/master/docs/dev/explain.md#generating-the-documentation to update the explain